src/Providers/AddonServiceProvider.php

```diff
@@ -298,8 +322,13 @@ protected function bootPolicies()
 
     protected function bootCommands()
     {
+        $commands = collect($this->commands)
+            ->merge($this->autoloadFilesFromFolder('Commands', Command::class))
```

Comment:
I've often put addon commands in `src/Console/Commands` to match where they'd live in an app scenario. I wonder if that's something we want to support here too?

Reply:
Added a search in that folder too